examples/jsm/renderers/webgl/nodes/WebGLNodeBuilder.js

```diff
@@ -460,29 +466,45 @@ class WebGLNodeBuilder extends NodeBuilder {
 
 		const uniforms = this.uniforms[ shaderStage ];
 
-		let snippet = '';
+		let output = '';
 
 		for ( const uniform of uniforms ) {
 
+			let snippet = null;
+
 			if ( uniform.type === 'texture' ) {
 
-				snippet += `uniform sampler2D ${uniform.name}; `;
+				snippet = `sampler2D ${uniform.name}; `;
 
 			} else if ( uniform.type === 'cubeTexture' ) {
 
-				snippet += `uniform samplerCube ${uniform.name}; `;
+				snippet = `samplerCube ${uniform.name}; `;
 
 			} else {
 
 				const vectorType = this.getVectorType( uniform.type );
 
-				snippet += `uniform ${vectorType} ${uniform.name}; `;
+				snippet = `${vectorType} ${uniform.name}; `;
+
+			}
+
+			const precision = uniform.node.precision;
+
+			if ( precision !== 'medium' ) {
```

Comment:
I think we should remove this check and always specify precision, even if it's mediump, like it's done on line 495 (https://github.com/mrdoob/three.js/pull/25561/files#r1116479196).